src/components/panel-control/PanelControl.js

```diff
@@ -48,6 +48,8 @@ const PanelControl = () => {
     });
   }, [layers]);
 
+  console.log(themeName);
```

Comment:
You should remove this console statement.

Reply:
Removed